test/unit/mocked_data/test_get_snmp_information/normal/show_snmp_contact.json

```diff
@@ -0,0 +1,3 @@
+{
+    "contact": "noc@cloudflare.com"
```

Comment:
On my device this is returned something like:

```
"contact": "\"noc@cloudflare.com\""
```

Which makes the output of this method:

```
u'contact': u'"noc@cloudflare.com"'
```

You only need to a strip those quotes when dumping the output - leaving inline comment where you need to apply changes.